impl/maven-cli/src/main/java/org/apache/maven/cling/invoker/CommonsCliOptions.java

```diff
@@ -472,24 +472,28 @@ public void displayHelp(String command, Consumer<String> pw) {
 
             pw.accept("");
 
-            StringWriter sw = new StringWriter();
-            PrintWriter pw2 = new PrintWriter(sw);
-            formatter.printHelp(
-                    pw2,
-                    width,
-                    commandLineSyntax(command),
-                    System.lineSeparator() + "Options:",
-                    options,
-                    HelpFormatter.DEFAULT_LEFT_PAD,
-                    HelpFormatter.DEFAULT_DESC_PAD,
-                    System.lineSeparator(),
-                    false);
-            pw2.flush();
-            for (String s : sw.toString().split(System.lineSeparator())) {
+            for (String s : getStringWriter(command, formatter, width).toString().split(System.lineSeparator())) {
                 pw.accept(s);
             }
         }
 
+        private StringWriter getStringWriter(String command, HelpFormatter formatter, int width) {
```

Comment:
```suggestion
        private StringWriter getFoo(String command, HelpFormatter formatter, int width) {
```
is this even a good name?

Reply:
No, this is not a getter. And the return type shouldn't be part of the method name. Something like makeMessage might be more approprriate. But I'm also not sure this change should be part of this PR since it's not about try with resources.